plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -340,9 +340,11 @@ const config: ControlPanelConfig = {
             config: {
               type: 'CheckboxControl',
               label: t('Show Min Label'),
-              default: xAxisShowMinLabel,
               renderTrigger: true,
               description: t('Show Min Label'),
+              mapStateToProps: () => ({
+                value: xAxisShowMinLabel,
```

Comment:
could we just use the following code without mapStateToProps?
```
{
    ....
    ....
    description: t('Show Min Label'),
    value: xAxisShowMinLabel,
}
```

Reply:
I tried. It does not work as expected.

Reply:
`mapStateToProps` means that "mapping state of the store to the property of the component".